sdk/core/AzureCore/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Release History
+
+## 1.0.0-beta.13 (Unreleased)
+### Bugs Fixed
+- Changed the format for application ID in `UserAgentPolicy` to remove the square brackets around it.  
+
```

Comment:
This should not be part of your PR. This is the change on main.

Reply:
reverted the cherry-pick, we won't worry about including this change in this particular patch